src/test/java/jalau/at18/katas/likes/daniela/LikesTest.java

```diff
@@ -0,0 +1,31 @@
+package jalau.at18.katas.likes.daniela;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class LikesTest {
+    @Test
+    public void testLikes() {
+        Like like = new Like();
+        String[] finalnamecase1 = {"Daniela", "Alejandra", "javier", "mayra"};
+        String[] finalnamecase2 = {"Daniela", "Alejandra", "javier"};
+        String[] finalnamecase3 = {"Daniela", "Alejandra"};
+        //String[] finalnamecase4 = {"Daniela"};
+        //String[] finalnamecase5 = {};
+        String inputnames1 = like.myLikes(finalnamecase1);
+        String inputnames2 = like.myLikes(finalnamecase2);
+        String inputnames3 = like.myLikes(finalnamecase3);
+        //String inputnames4 = like.myLikes(finalnamecase4);
+        //String inputnames5 = like.myLikes(finalnamecase5);
+        String output1 = "Daniela ,Alejandra and 2 others likes this";
```

Comment:
There is an unexpected space between "Daniela" and ",", and there is a missing space between "," and "Alejandra". This is defined in the kata requirement:
```
["Alex", "Jacob", "Mark", "Max"]  -->  "Alex, Jacob and 2 others like this"
```
Please update it so it matches kata requirement. The other verifications look good